gravitee-apim-console-webui/src/management/settings/groups/group/group.html

```diff
@@ -204,11 +204,12 @@ <h2>Dependents</h2>
               <table md-table ng-init="orderGrp = 'displayName'">
                 <thead md-head md-order="orderGrp">
                   <tr md-row>
-                    <th md-column width="22%" md-order-by="displayName">Name</th>
-                    <th md-column width="22%">Group Admin</th>
-                    <th md-column width="22%">API Role</th>
-                    <th md-column width="22%">Application Role</th>
-                    <th md-column width="12%"></th>
+                    <th md-column width="18%" md-order-by="displayName">Name</th>
+                    <th md-column width="18%">Group Admin</th>
+                    <th md-column width="18%">API Role</th>
+                    <th md-column width="18%">Application Role</th>
+                    <th md-column width="18%">Integration Role</th>
+                    <th md-column width="10%"></th>
```

Comment:
Maybe we can put the width with a flex property(probably need tr should be display flex)? With 1 for all and 0.1 for last and this will be same whatever the number of columns?